algorithms/Python/recursion/n-th_fibonacci_number.py

```diff
@@ -0,0 +1,16 @@
+# Function for nth fibonacci number - Using recursion
+# Taking 1st two fibonacci numbers as 0 and 1
+ 
+FibArray = [0, 1]
+def fibonacci(n):
+    if n<0:
+        print("Incorrect input")
+    elif n<= len(FibArray):
```

Comment:
What will happen when `n` is 0 ?

Reply:
the function will return 0

Reply:
Which will make your series look something like :

`1 ,0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, ...`

but shouldn't it ideally look like following ?

`0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, ...`

Reply:
you're right, how can I fix this?

Reply:
Just create an extra case ( using `elif` ) to handle the situation when `n` is 0.

Reply:
It won't work, actually my code is optimized for bigger numbers, Like it can return unto n = 999. I hope it wont be a problem.

Reply:
okay, please check now.